frame/collective/src/lib.rs

```diff
@@ -453,11 +319,10 @@ decl_module! {
 		/// - 1 event
 		/// # </weight>
 		#[weight = (
-			weight_for::execute::<T, I>(
-				MAX_MEMBERS.into(),
-				proposal.get_dispatch_info().weight,
-				*length_bound as Weight,
-			),
+			T::WeightInfo::execute(
+				*length_bound as u32, // B
```

Comment:
in case length_bound is somehow moved to u64, this is safer
```suggestion
				*length_bound, // B
```